metadata.json

```diff
@@ -11,15 +11,15 @@
   "dependencies": [
     {
       "name": "puppet/archive",
-      "version_requirement": ">= 1.0.0 < 6.0.0"
+      "version_requirement": ">= 1.0.0 < 5.0.0"
```

Comment:
puppet/mysql_java_connector does not support 5.x

Reply:
I'll start working on modulesync issues in puppet/mysql_java_connector so that we can get this inline as well.

Reply:
see https://github.com/voxpupuli/puppet-mysql_java_connector/pull/77

Reply:
thanks for all the work. I made a new release for the mysql_java_connector.